### PR TITLE
Feature/instanceviz

### PR DESCRIPTION
**Pull request by Roman-Maksimov**

### CODE REVIEW

File: src/extensions/react-flow/edges/EdgeDefault.tsx [hunk 1]
```diff
@@ -11,6 +11,9 @@ import { drawEdgeStraight} from "./utils";
 
 export interface EdgeDefaultDataProps {
     pathGlowWidth?: number;
+    inversePath?: boolean;
```
Comment (haschek) [inline src/extensions/react-flow/edges/EdgeDefault.tsx:14]: Please extend (or add) an Storybook example to demonstrate this feature.

File: src/extensions/react-flow/edges/EdgeDefault.tsx [hunk 2]
```diff
@@ -11,6 +11,9 @@ import { drawEdgeStraight} from "./utils";
...
+    markerStart?: string;
+    renderLabel?: (edgeCenter: [number, number, number, number]) => React.ReactNode;
```
Comment (haschek) [inline src/extensions/react-flow/edges/EdgeDefault.tsx:16]: Please extend (or add) an Storybook example to demonstrate this feature.

**Review by haschek — CHANGES_REQUESTED** [review]

File: src/extensions/react-flow/markers/MarkerArrowClosedInverse.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React, { FC } from "react";
+
+export const MarkerArrowClosedInverse: FC = () => {
```
Comment (andreas-schultz) [inline src/extensions/react-flow/markers/MarkerArrowClosedInverse.tsx:3]: Please add a comment to public/exported components.

File: src/extensions/react-flow/markers/ReactFlowMarkers.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React, { FC } from "react";
+
+import { MarkerArrowClosedInverse } from "./MarkerArrowClosedInverse";
+
+export const ReactFlowMarkers: FC = () => {
```
Comment (andreas-schultz) [inline src/extensions/react-flow/markers/ReactFlowMarkers.tsx:5]: Same as above.

File: src/extensions/react-flow/markers/MarkerArrowClosedInverse.tsx (hunk 3 above)
Comment (haschek) [reply src/extensions/react-flow/markers/MarkerArrowClosedInverse.tsx:3]: There will be an new issue to clean a few things up related to this brunch. I will add this request there.

**Pull request merged**